samples/batch/transcription-enabled-storage/Connector/DatabaseConnector.cs

```diff
@@ -0,0 +1,254 @@
+﻿// <copyright file="DatabaseConnector.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data.SqlClient;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Logging;
+
+    public class DatabaseConnector : IDisposable
+    {
+        private ILogger Logger;
+
+        private string DBConnectionString;
+
+        private SqlConnection Connection;
+
+        public DatabaseConnector(ILogger logger, string dbConnectionString)
+        {
+            Logger = logger;
+            DBConnectionString = dbConnectionString;
+        }
+
+        public async Task<bool> StoreTranscriptionAsync(
+            Guid transcriptionId,
+            string locale,
+            string fileName,
+            float approximateCost,
+            SpeechTranscript speechTranscript)
+        {
+            if (speechTranscript == null)
+            {
+                throw new ArgumentNullException(nameof(speechTranscript));
+            }
+
+            try
+            {
+                Connection = new SqlConnection(DBConnectionString);
+                Connection.Open();
+
+                var query = "INSERT INTO dbo.Transcriptions (ID, Locale, Name, Source, Timestamp, Duration, NumberOfChannels, ApproximateCost)" +
+                    " VALUES (@id, @locale, @name, @source, @timestamp, @duration, @numberOfChannels, @approximateCost)";
+
+                using (var command = new SqlCommand(query, Connection))
+                {
+                    command.Parameters.AddWithValue("@id", transcriptionId);
+                    command.Parameters.AddWithValue("@locale", locale);
+                    command.Parameters.AddWithValue("@name", fileName);
+                    command.Parameters.AddWithValue("@source", speechTranscript.Source);
+                    command.Parameters.AddWithValue("@timestamp", speechTranscript.Timestamp);
+                    command.Parameters.AddWithValue("@duration", speechTranscript.Duration);
+                    command.Parameters.AddWithValue("@numberOfChannels", speechTranscript.CombinedRecognizedPhrases.Count());
+                    command.Parameters.AddWithValue("@approximateCost", approximateCost);
+
+                    var result = await command.ExecuteNonQueryAsync().ConfigureAwait(false);
+
+                    if (result < 0)
+                    {
+                        Logger.LogInformation("Did not store json in Db, command did not update table");
+                    }
+                    else
+                    {
+                        var phrasesByChannel = speechTranscript.RecognizedPhrases.GroupBy(t => t.Channel);
```

Comment:
RecognizedPhrases is combinedResult or List of SegmentResults?

Reply:
as discussed: V3 uses RecognizedPhrases instead of SegmentResults